app/javascript/controllers/media_tag_controller.js

```diff
@@ -45,8 +45,8 @@ export default class extends Controller {
         window.dispatchEvent(new CustomEvent('auth-stanford-restricted'))
       window.dispatchEvent(new CustomEvent('auth-success'))
     } else {
-      const event = new CustomEvent('auth-denied', { detail: result.authResponse })
-      window.dispatchEvent(event)
+        const event = new CustomEvent('auth-denied', { detail: result.authResponse })
+        window.dispatchEvent(event)
```

Comment:
Revert extra spaces?

Reply:
Those aren't extra.  Doesn't JS world use 4 spaces?

Reply:
The rest of the file, and other JS files above, looks like 2 spaces. Smells like a JS file in a Ruby shop. :)

Reply:
You're not Rubocop! You can't tell me what to do! 😆